include/libswiftnav/ephemeris.h

```diff
@@ -39,6 +42,34 @@ typedef struct {
 } ephemeris_xyz_t;
 
 typedef struct {
+  /*u8 m;*/     /* string number */
+  /*u8 B;*/     /* unhealthy SV indication, should be stored in ephemeris_t */
+  /*u8 n;*/     /* SV id , should be stored in ephemeris_t*/
+  /*u8 l;*/     /* unhealthy SV indication, should be stored in ephemeris_t */
+  /*u8 P;*/     /* SV operation mode in respect of time parameters TODO: decide is we need to store it */
+
+  u8 tk_hours;            /* Current frame start time since beginning of day */
+  u8 tk_minutes;          /* Not needed in ephemeris. TODO: decide if it needs to be stored here. */
+  u8 tk_half_minutes;
+
+  u16 tb;                 /* Index of the time interval within current day */
+  u8 sv_type;             /* Coded as word M in broadcasted ephemeris */
+  double gamma;
+  double tau;
+  double pos[3];          /* Position of the SV at tb in PZ-90.02 coordinates system, km */
```

Comment:
Didn't quite get that. Do you mean using ephemeris_xyz_t in ephemeris_glo_t? Or one more struct to be used in ephemeris_xyz_t and in ephemeris_glo_t? I don't like either of those as the first adds unnecessary fields to GLO ephe and the second requires quite a bit of changes in the code, in all places accessing those fields. So, I'll leave it as is for now, let's discuss. I change units to meters, though.